lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py

```diff
@@ -39,16 +38,13 @@ def setUp(self):
 
     def test_services_are_running(self):
 
-        if self._s1ap_wrapper.magmad_util.init_system.value != InitMode.SYSTEMD.value:
-            self.skipTest("Systemd only test.")
```

Comment:
Do we want to run the test in this form for the docker setup? (Not a docker expert, but ...) iiuc then docker/docker-compose has a way better health check then systemd, i.e., multiple runs in order to identify a re-start loop is not necessary.

Reply:
This is of course something to be debated. Maybe I remove the docker support for this test out of this PR again and just add the service function and adapt the test to it.